### DISCUSSION TITLE
Hello I am using Ollama3 and I am wondering about the Prompts section in settings.

### DISCUSSION THREAD

**Discussion by ObliviousPawn** [longy2k/obsidian-bmo-chatbot#102] [Q&A]
![ShareX_rpOi819jfO](https://github.com/user-attachments/assets/c7df1266-6b1f-41b4-a3b5-733b6be20f7a)
Can you give a little more clarification on how this works, and how to make it work. I've seen other applications that you can put in a prompt in a certain section to get the LLM to answer or act in a certain way but I am not sure if this is the same thing?

**Comment by ObliviousPawn** [comment]
Never mind I just saw that it is the profiles section I am looking for thank you.

**Reply by longy2k** [reply]
If you create a "Prompts" folder and add your markdown notes in, you can /prompt to get a list of the markdown notes. Then you can insert whatever prompts you like :)

**Reply by ObliviousPawn** [reply]
Thank you =)

**Reply by gerroon** [reply]
Is it possible to get some prompts sample files? thanks
